src/identity/components/Avatar.tsx

```diff
@@ -1,13 +1,18 @@
+import type { Address } from 'viem';
+import { useEffect, useState } from 'react';
+import { base } from 'viem/chains';
+import { getEASAttestations } from '../getEASAttestations';
 import { useAvatar } from '../hooks/useAvatar';
 import { useName } from '../hooks/useName';
-import type { Address } from 'viem';
+import { WithAvatarBadge } from './WithAvatarBadge';
 
 type AvatarProps = {
   address: Address;
   className?: string;
   loadingComponent?: JSX.Element;
   defaultComponent?: JSX.Element;
   props?: React.ImgHTMLAttributes<HTMLImageElement>;
+  attest?: boolean;
```

Comment:
Maybe `showAttestationBadge` or `displayAttestationBadge` is better

verb + utility + component

@kyhyco what do you think?

Reply:
changed to `attest`

Reply:
`attest` feels even more confusing....

Reply:
`showAttestation`?